DiscordChatExporter.Core.Rendering/Resources/HtmlCore.css

```diff
@@ -57,6 +57,70 @@ img {
     border-radius: 3px;
 }
 
+.spoiler.hidden {
+    cursor: pointer;
+}
+
+.spoiler.hidden .spoilerText {
```

Comment:
Rename `.spoilerText` to `.spoiler-text` for consistency.

Reply:
Should it not be `.spoiler__text` then, since it's a child element depending on `.spoiler`? (This is my first time hearing of BEM notation, maybe I'm misunderstanding it.)

Reply:
Not sure either, but I think `.spoiler-text` would be more consistent with the rest of the class names. I care more about consistency as opposed to correctness here. :)